inst/include/interface/rcpp/rcpp_interface.hpp

```diff
@@ -318,151 +318,246 @@ void clear() {
+  Rcpp::function("clear_error_log", clear_error_log,
+                 "Clears the contents of error log file. No inputs are required.");
+  Rcpp::function("clear_data_log", clear_data_log,
+                 "Clears the contents of data log file. No inputs are required.");
+  Rcpp::function("clear_population_log", clear_population_log,
+                 "Clears the contents of population log file. No inputs are required.");
+  Rcpp::function("clear_model_log", clear_model_log,
+                 "Clears the contents of model log file. No inputs are required.");
   Rcpp::function("clear_recruitment_log", clear_recruitment_log);
-  Rcpp::function("clear_fleet_log", clear_fleet_log);
+  Rcpp::function("clear_fleet_log", clear_fleet_log,
+                 "Clears the contents of fleet log file. No inputs are required.");
   Rcpp::function("clear_growth_log", clear_growth_log);
-  Rcpp::function("clear_maturity_log", clear_maturity_log);
-  Rcpp::function("clear_selectivity_log", clear_selectivity_log);
-  Rcpp::function("clear_debug_log", clear_debug_log);
+  Rcpp::function("clear_maturity_log", clear_maturity_log,
+                 "Clears the contents of maturity log file. No inputs are required.");
+  Rcpp::function("clear_selectivity_log", clear_selectivity_log,
+                 "Clears the contents of selectivity log file. No inputs are required.");
+  Rcpp::function("clear_debug_log", clear_debug_log,
+                 "Clears the contents of debug log file. No inputs are required.");
 
   Rcpp::class_<Parameter>("Parameter")
       .constructor()
       .constructor<double>()
       .constructor<Parameter>()
-      .field("value", &Parameter::value_m)
-      .field("min", &Parameter::min_m)
-      .field("max", &Parameter::max_m)
-      .field("is_random_effect", &Parameter::is_random_effect_m)
-      .field("estimated", &Parameter::estimated_m);
+      .field("value", &Parameter::value_m,
+             "Initial value of the parameter.")
+      .field("min", &Parameter::min_m,
+             "Min value of the parameter.")
+      .field("max", &Parameter::max_m,
+             "Max value of the parameter.")
+      .field("is_random_effect", &Parameter::is_random_effect_m,
+             "Is the parameter a random effect parameter? Default value is false.")
+      .field("estimated", &Parameter::estimated_m,
+             "Is the parameter estimated? Default value is false.");
 
   Rcpp::class_<BevertonHoltRecruitmentInterface>("BevertonHoltRecruitment")
       .constructor()
-      .field("logit_steep", &BevertonHoltRecruitmentInterface::logit_steep)
-      .field("log_rzero", &BevertonHoltRecruitmentInterface::log_rzero)
-      .field("log_devs", &BevertonHoltRecruitmentInterface::log_devs)
+      .field("logit_steep", &BevertonHoltRecruitmentInterface::logit_steep,
+             "Recruitment relative to unfished recruitment at 20% of unfished spawning biomass. Should be a value between 0.2 and 1.0.")
```

Comment:
It might be good to spell out percent rather than using the sign just in case there are downstream effects, for example within latex it would need to be escaped.

Reply:
Done.